lib/generators/shopify_app/add_gdpr_jobs/templates/customer_data_request_job.rb.tt

```diff
@@ -0,0 +1,22 @@
+class CustomersDataRequestJob < ActiveJob::Base
+  extend ShopifyAPI::Webhooks::Handler
+
+  class << self
+    def handle(topic:, shop:, body:)
+      perform_later(topic: topic, shop_domain: shop, webhook: body)
+    end
+  end
+
+  def perform(topic:, shop_domain:, webhook:)
+    shop = Shop.find_by(shopify_domain: shop_domain)
+
+    if shop.nil?
+      logger.error("#{self.class} failed: cannot find shop with domain '#{shop_domain}'")
+      return
+    end
+
+    shop.with_shopify_session do
+      logger.info("#{self.class} started for shop '#{shop_domain}'")
```

Comment:
Im not sure if anyone has any context on why this log message is here but not in the others?